Servers/WindowServer/Window.cpp

```diff
@@ -420,4 +420,19 @@ void Window::detach_client(Badge<ClientConnection>)
     m_client = nullptr;
 }
 
+void Window::recalculate_rect()
+{
+    if (is_resizable()) {
```

Comment:
Let's use early return style :)

```suggestion
    if (!is_resizable())
        return;
```